integration-tests/pom.xml

```diff
@@ -243,6 +243,7 @@
                 <module>maven</module>
                 <module>scala</module>
                 <module>kotlin</module>
+                <module>kotlin-reflection-native</module>
                 <module>kotlin-serialization</module>
```

Comment:
Does it need to be a new module?

Asking because each module adds time to CI because it needs to be built in native.

If it can't, be added to an existing module, we'll need to have this configured in the JSON file used by CI for native tests

Reply:
originally i had it as just another set of tests in the kotlin integration test but the other tests are all bound to the `integration-test` and `verify` phases.  No matter what I tried, the native tests ran all the time.  So I finally moved them out to their own tests so I could have full control over the phases involved.  But you're right about the native test config.  I'll update that file.

Reply:
I'll have a look tomorrow, because we really like to avoid adding new modules

Reply:
agreed.  i'd love to know what i was missing but i banged away at for a while before finally giving up.  It could be that the tests don't actually need to be bound explicitly to the phases they are but it might be something in how maven tests are run against itself that dictate that.

Reply:
I would just add these new tests to the `quarkus-integration-test-resteasy-reactive-kotlin-standard` module.